src/client.ts

```diff
@@ -2602,10 +2602,13 @@ export class StreamChat<StreamChatGenerics extends ExtendableGenerics = DefaultG
     );
   }
 
-  async getMessage(messageID: string) {
-    return await this.get<GetMessageAPIResponse<StreamChatGenerics>>(
-      this.baseURL + `/messages/${encodeURIComponent(messageID)}`,
-    );
+  async getMessage(messageID: string, show_deleted_message?: boolean) {
```

Comment:
Instead of a single parameter, I would add it as an options object:
`options: GetMessageOptions`
```suggestion
  async getMessage(messageID: string, options?: GetMessageOptions) {
  
  type GetMessageOptions = {
    show_deleted_message?: boolean
  }
```

This way it's more easily extendable in the future in case you want to add another option / parameter

Reply:
Yes nice one! Done